src/reference/asciidoc/amqp.adoc

```diff
@@ -385,6 +385,10 @@ Starting with Spring AMQP _version 1.5_, these properties can be set directly on
 If both discrete properties and `sslPropertiesLocation` is provided, properties in the latter will override the
 discrete values.
 
+Starting with _version 1.5.6_, the factory bean exposes a property `customClientProperties` which allows you to add
+properties to connections.
+These properties appear on the RabbitMQ Admin UI when viewing connections.
```

Comment:
These sentences don't make sense any more, since we don't introduce any new option.